src/Controls/src/Core/Shapes/Line.cs

```diff
@@ -38,5 +52,15 @@ public double Y2
 			set { SetValue(Y2Property, value); }
 			get { return (double)GetValue(Y2Property); }
 		}
+
+		public PathF PathForBounds(Graphics.Rectangle rect)
+		{
+			var path = new PathF();
+
+			path.MoveTo((float)X1, (float)Y1);
+			path.LineTo((float)X2, (float)Y2);
+
+			return path.AsScaledPath((float)Width / (float)rect.Width);
```

Comment:
Since Line has a Width, and start/end points are given for specific values, we need to assume the Path is created in the bounds of the View of Line, and then scale it appropriately for the given rectangle.  Is this correct?